setup.cfg

```diff
@@ -99,6 +99,7 @@ tests =
 
 pandas =
     pandas>=1.5,<3.0
+    numpy<2.0.0
```

Comment:
My only comment before this is ready for review:

Is there any way we can expand this out? Possibly adding in some logic we can remove later on where it does X if numpy < 2.0.0, and Y if numpy >= 2.0.0?


My concern is that as we add more of these constraints we'll run into compatibility issues with others using this package.

Reply:
I had the same concern. I will look more into this and post updates here. For context, we are experiencing the same problem as this [issue](https://github.com/numpy/numpy/issues/26710). The dev who raised the issue said that they are doing [this](https://github.com/numpy/numpy/issues/26710#issuecomment-2171874281) as a solution.

Reply:
But it's more complicated because the newer Pandas versions aren't compatible with python 3.8, So we need the pandas version constraints to remain the same I think. I'm not sure there is anything better to do other than keeping this numpy constraint until we no longer support python 3.8

Reply:
^ Can we create a ticket for that, we also should add support for python 3.12, but i already know there are going to be issues with that since people have encountered issues when using python 3.12 with the python client.

Reply:
https://sagebionetworks.jira.com/browse/SYNPY-1517